### PR TITLE
Add capabilities error

### LINKED ISSUE #3289: v5 Test Runner reports no specs found to run when no capabilities property set
**Environment (please complete the following information):**
 - **WebdriverIO version:** 5.2.7
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** async
 - **Node.js version:** 8.11.2
 - **NPM version:** 5.8.0

**Describe the bug**
I used the configuration file that was generated for me using the WDIO CLI. I then commented out the capabilities section.

**Expected behavior**
I expected that an error telling me clearly I am missing a capabilities section, and will not run my tests. Instead I got the error message:

`2019-01-09T20:32:39.686Z ERROR wdio-cli:Launcher: No specs found to run, exiting with failure`

which makes me think the pattern in my specs property is wrong.

### PR DESCRIPTION
**Pull request by erwinheitzman**
## Proposed changes

Add check for missing capabilities. Fixes #3289 

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-cli/src/launcher.js [hunk 1]
```diff
@@ -102,6 +102,15 @@ class Launcher {
             })
         }
 
+        /**
+         * fail if no caps were found
+         */
+        if (!caps.length) {
```
Comment (abjerstedt) [inline packages/wdio-cli/src/launcher.js]: We should add a UT for this

Comment (erwinheitzman) [reply packages/wdio-cli/src/launcher.js]: Can you explain what you mean by a UT? ;)

Comment (abjerstedt) [reply packages/wdio-cli/src/launcher.js]: Unit Test.

Comment (abjerstedt) [reply packages/wdio-cli/src/launcher.js]: Unit Test - If this is a new check, we should test it.

**Comment by christian-bromann** [comment]
ping @erwinheitzman

**Pull request closed without merging**